composer.json

```diff
@@ -14,15 +14,16 @@
         "rmccue/requests": "~1.6",
         "symfony/finder": ">2.7",
         "wp-cli/mustangostang-spyc": "^0.6.3",
-        "wp-cli/php-cli-tools": "~0.11.2"
+        "wp-cli/php-cli-tools": "~0.11.2",
+        "wp-cli/wp-cli-tests": "^2.0.7"
```

Comment:
This is a duplicate and needs to be removed (as it is also in the wrong section).

Reply:
fixed in https://github.com/wp-cli/wp-cli/pull/4906/commits/4a782e0b8822dcd941c7812fb0956cd0bec2545e

Reply:
No, the line is still there. https://github.com/wp-cli/wp-cli/commit/4a782e0b8822dcd941c7812fb0956cd0bec2545e only fixed the indentation problem.